packages/catalog-process/test/catalogService.integration.test.ts

```diff
@@ -4655,7 +4657,50 @@ describe("database test", async () => {
           )
         ).rejects.toThrowError(tenantKindNotFound(producer.id));
       });
+      it("should throw riskAnalysisDuplicate if risk analysis name is duplicated", async () => {
+        const producerTenantKind: TenantKind = TenantKind.Enum.PA;
```

Comment:
```suggestion
        const producerTenantKind: TenantKind = randomArrayItem(
          Object.values(tenantKind)
        );
```
We can randomize this too